cmd/installer/pkg/install/errata.go

```diff
@@ -101,3 +103,28 @@ func upgradeFromPreIfnamesTalos() (bool, error) {
 
 	return talosVersion.DisablePredictableNetworkInterfaces(), nil
 }
+
+// SanitizeKernelArgs removes kernel arguments that are not needed for the installer.
+func SanitizeKernelArgs(cmdline []string) []string {
+	kernelArgsToDrop := xslices.Filter(cmdline, func(s string) bool {
+		// it's valid to have a kernel arg just as `-`.
+		// Ref: https://www.kernel.org/doc/html/v4.14/admin-guide/kernel-parameters.html
+		return len(s) > 1 && s[0] == '-'
+	})
+
+	return xslices.Filter(cmdline, func(s string) bool {
```

Comment:
would it make more sense to put this into the `cmdline.AppendAll` code?

Reply:
yeh, that's fine, I was not sure if we should do that there